guide/payments/send.md

```diff
@@ -0,0 +1,161 @@
+
+##### Fee Estimation
+Fee estimations are largely inaccurate — this is because the fee rate is impacted by everyone else who are trying to get their transactions into a block. The rate is constantly changing so it is difficult to make an accurate prediction.
+
+The fee recommendation in your application can cause senders to end up over paying in fees, or waiting long periods of time to get their transactions confirmed.
+
+### Setting the transaction fee manually
+
+![](https://i.imgur.com/LcEGYjG.png)
+
+When allowing users to set their own fee, it is important to clearly communicate the estimated cost and confirmation time, along with some details on how the fee gets calculated ([see below](#How-are-transaction-fees-calculated)).
+
+Human error with fee selection can lead to [costly mistakes](https://www.coindesk.com/dumb-mistakes-costly-bitcoin-losses). Someone can set a fee rate that's way too high, overpaying in fees or having transactions stuck for long periods of time if the fee is too low. For these reasons you should carefully consider if and how to expose transaction fees to users depending on the use case.
+
+**Do's**
+
+- Clearly communicate information related to the fee rate options
+- Display the fee rate and estimated transaction confirmation time
+- Display bitcoin and local currency denominations for cost reference
+- Include an option to adjust fee rate based on some presets (e.g. high, medium, low priority)
+- Display a warning if the user enters an amount far beyond the recommended rate
+- Consider offering a [replace by fee]() option to speed up stuck transactions
+
+**Don'ts**
+
+- Allow for confusion between total fee and fee rate
+- Allow users to overpay in transaction fees
+
+## Reviewing and Approving the Payment
+
+A valid transaction that is broadcasted to the network cannot be reversed, so it is critical that the sender is given a chance to double check the payment details (amount, recipient address, total fee, etc) before submitting the transaction.
+
+![](https://i.imgur.com/mk3pJHa.png)
+
+**Do's**
+
+- Allow users to review payment details and adjust if necessary, before submitting the transaction
+- Show amount and fee in both bitcoin and local currency denominations
+- If the wallet allows users to set a spending limit and the current transaction exceeds it, make sure users go through some security check (biometric, enter PIN, 2FA password, etc). This can also be used if the 100% of the funds are being sent. 
+
+
+**Don'ts**
+
+- Assume the user has got all the details correct, give them the chance to look over the information before broadcasting the transaction
+
+## Transaction processing and confirmation
```

Comment:
I've commented in #173 that processing and confirmation is a bit vague, this in a way compensates for it.